src/openrct2/world/map.c

```diff
@@ -2728,10 +2728,13 @@ void game_command_place_large_scenery(sint32* eax, sint32* ebx, sint32* ecx, sin
         rct_tile_element* tile_element = map_get_surface_element_at(curTile.x / 32, curTile.y / 32);
         if(tile_element != NULL){
             sint32 height = tile_element->base_height * 8;
+            sint32 type = scenery_large_get_type(tile_element);
 
-            if(tile_element->properties.scenerymultiple.type & 0xF){
+            if (type & 0xF)
```

Comment:
How odd this code is. Why would a type also encode height info. I want to double check the assembly for this section. Need to update wiki as well about this info.

Reply:
Oh, I think I know!
It gets `tile_element` by doing `map_get_surface_element_at()`. So this isn't a large scenery object at all.It's a surface element, and the property at this location is `slope`.

Reply:
https://github.com/OpenRCT2/OpenRCT2/pull/6684/files